Blockzilla/UIConstants.swift

```diff
@@ -291,6 +294,7 @@ struct UIConstants {
         static let externalAppLinkWithAppName = NSLocalizedString("externalAppLinkWithAppName.messageTitle", value: "%@ wants to open %@", comment: "Dialog title used for opening an external app from Focus. First placeholder string is the app name of either Focus or Klar and the second placeholder string specifies the app it wants to open.")
         static let open = NSLocalizedString("ExternalAppLink.openTitle", value: "Open", comment: "Button label for opening another app from Focus")
         static let cancel = NSLocalizedString("ExternalAppLink.cancelTitle", value: "Cancel", comment: "Button label used for cancelling to open another app from Focus")
+        static let close = NSLocalizedString("Menu.Close", value: "Close", comment: "Button label used to close a menu.")
```

Comment:
This will _likely_ need to be a bit more descriptive for when localization looks at it. They'll need more context about which menu this is that it's closing.